Py4GWCoreLib/enums.py

```diff
@@ -19,15 +21,21 @@ class SharedCommandType(IntEnum):
     MerchantMaterials = 15
     DisableHeroAI = 16
     EnableHeroAI = 17
-    LootEx = 18 #privately Handled Command, by Frenkey
+    LootEx = 18  # privately Handled Command, by Frenkey
+
+
```

Comment:
This is the main class I added here on this file. 

Everything else is auto-formatted by my lint rules 😓 

Let me know if this is an issue, and I can just revert. No functional code should be changed besides this.